grails-app/assets/javascripts/species.show.js

```diff
@@ -256,6 +256,7 @@ function loadExternalSources() {
                     } else {
                         $description.find('.content').html(dataObject.description);
                     }
+                    $description.find('img').addClass('img-responsive');
```

Comment:
That's a very peculiar way of adding style.

Reply:
i'd think that i'd do a better job if the content weren't pre-rendered